cli/command/system/testdata/docker-info-no-swarm.golden

```diff
@@ -1,3 +1,9 @@
+Client
+------
+Debug Mode: true
+
+Daemon
+------
```

Comment:
Instead of a separator-line, should we use the same format as on `docker version`?

```
Client: Docker Engine - Community
  Debug Mode: true

Server: Docker Engine - Community
  Containers: 0
    Running: 0
    Paused: 0

....
```

Reply:
I was just trying to avoid touching every single line ;-) (there's also a small possibility of breaking scripts, but they should be using the machine readable form anyway).

I'll make the indentation change and look into adding the platform version -- not sure how tricky that is to get hold of here.